zulip_bots/zulip_bots/bots/jira/jira.py

```diff
@@ -127,7 +163,40 @@ def initialize(self, bot_handler: Any) -> None:
             raise KeyError('No `domain` was specified')
 
         self.auth = make_jira_auth(username, password)
-        self.domain_with_protocol = 'https://' + domain
+
+        # Allow users to override the HTTP scheme
+        if re.match(r'^https?://', domain, re.IGNORECASE):
+            self.domain_with_protocol = domain
+        else:
+            self.domain_with_protocol = 'https://' + domain
+
+        # Use the front facing URL in output
+        self.display_url = config.get('display_url')
+        if not self.display_url:
+            self.display_url = self.domain_with_protocol
+
+    def jql_search(self, jql_query:str) -> str:
+        UNKNOWN_VAL = '*unknown*'
+        jira_response = requests.get(
+            f'{self.domain_with_protocol}/rest/api/2/search?jql={jql_query}&fields=key,summary,status',
```

Comment:
Unfortunately, we cannot use f-strings yet. We have to support python 3.5 as well, thus, we should use `"{}".format()` for now.